website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -240,22 +261,36 @@ metadata:
 ## Consul to Kubernetes
 
 This syncs Consul services into first-class Kubernetes services.
-Each Consul service is synced to an
-[ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
-service in Kubernetes. The external name is configured to be the Consul
-DNS entry.
+An [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+`Service` is created for each Consul service. The "external name" will be
```

Comment:
Clarify who or what does the creating by putting this sentence into active voice?